4.classify-features/4.classify-features.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+jupyter nbconvert --to python classify-features.ipynb
+python classify-features.py
```

Comment:
You keep mentioning awesome packages that I have never heard of before! I don't think it would match the workflow for this repository but I will definitely keep this in mind for future work.